mnc/control.py

```diff
@@ -183,10 +182,6 @@ def status_fengine(self, poll=False):
         for snap2name in snap2names:
             snap2num = int(snap2name.lstrip('snap'))
 
-            if poll:
-                lwa_fe = snap2_feng_etcd_client.Snap2FengineEtcdClient(snap2name, int(snap2name.lstrip('snap')))
-                lwa_fe.poll_stats()
-
             dd = ls.get_dict(f'/mon/snap/{snap2num}')
```

Comment:
Per the emaiI just sent you, this should always be a two-digit SNAP number (i.e. 01, 02, ... 10, 11)